CMakeLists.txt

```
@@ -25,36 +25,36 @@ project(OpenTimelineIO VERSION ${OTIO_VERSION} LANGUAGES C CXX)
 # maintenance and troubleshooting
 
 # Installation options
-option(OTIO_CXX_INSTALL    "Install the cpp bindings" ON)
-option(OTIO_PYTHON_INSTALL "Install the python bindings" ON)
+option(OTIO_CXX_INSTALL          "Install the C++ bindings" ON)
+option(OTIO_PYTHON_INSTALL       "Install the Python bindings" ON)
 option(OTIO_DEPENDENCIES_INSTALL "Install OTIO's C++ header dependencies (any and nonstd)" ON)
 option(OTIO_INSTALL_COMMANDLINE_TOOLS "Install the OTIO command line tools" ON)
 set(OTIO_PYTHON_INSTALL_DIR "" CACHE STRING "Python installation dir (such as the site-packages dir)")
 
 # Build options
-option(OTIO_SHARED_LIBS  "Build shared if ON, static if OFF" ON)
-option(OTIO_CXX_COVERAGE "Invoke code coverage if gcov is available" OFF)
+option(OTIO_SHARED_LIBS          "Build shared if ON, static if OFF" ON)
+option(OTIO_CXX_COVERAGE         "Invoke code coverage if gcov is available" OFF)
 option(OTIO_AUTOMATIC_SUBMODULES "Fetch submodules automatically" ON)
 
 #------------------------------------------------------------------------------
 # Set option dependent variables
 
 if(OTIO_PYTHON_INSTALL)
-    # reconcile install directories for builds incorporating python in order
+    # reconcile install directories for builds incorporating Python in order
     # that default behaviors match a reasonable expectation, as follows:
     #
-    # if nothing has been set, 
+    # if nothing has been set,
     #   Python: ${Python_SITEARCH}/opentimelineio
     #   C++:    ${Python_SITEARCH}/opentimelineio/cxx-libs
+    # if only OTIO_PYTHON_INSTALL_DIR has been set,
+    #   Python: ${OTIO_PYTHON_INSTALL_DIR}/opentimelineio
+    #   C++:    ${OTIO_PYTHON_INSTALL_DIR}/opentimelineio/lib
     # if only CMAKE_INSTALL_PREFIX has been set,
     #   Python: ${CMAKE_INSTALL_PREFIX}/opentimelineio/python
     #   C++:    ${CMAKE_INSTALL_PREFIX}/opentimelineio
```

From the code:
```cmake
set(OTIO_RESOLVED_PYTHON_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/python")
set(OTIO_RESOLVED_CXX_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}")
```

There's definitely no `opentimelineio` folder between the prefix and `/python`, but not sure if below change is correct either. Is a subfolder created so that the Python path ends up as `${CMAKE_INSTALL_PREFIX}/python/opentimelineio`?

```suggestion
    # if only CMAKE_INSTALL_PREFIX has been set,
    #   Python: ${CMAKE_INSTALL_PREFIX}/python
    #   C++:    ${CMAKE_INSTALL_PREFIX}/
```

---

The C++ part seems to be broken. The .pyd, .lib and .dll files end up in `C:\opentimelineio`. The prefix is apparently not used for these files:

```
cmake --install .
-- Install configuration: "Release"
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/deps/any/any.hpp
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/deps/nonstd/optional.hpp
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentime/errorStatus.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentime/rationalTime.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentime/stringPrintf.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentime/timeRange.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentime/timeTransform.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentime/version.h
-- Installing: /opentimelineio/opentime.lib
-- Installing: /opentimelineio/opentime.dll
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/OpenTimeConfig.cmake
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/OpenTimeConfig-release.cmake
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/any.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/anyDictionary.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/anyVector.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/clip.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/composable.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/composition.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/deserialization.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/effect.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/errorStatus.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/externalReference.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/freezeFrame.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/gap.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/generatorReference.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/imageSequenceReference.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/item.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/linearTimeWarp.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/marker.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/mediaReference.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/missingReference.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/optional.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/safely_typed_any.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/serializableCollection.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/serializableObject.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/serializableObjectWithMetadata.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/serialization.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/stack.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/stackAlgorithm.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/stringUtils.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/timeEffect.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/timeline.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/track.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/trackAlgorithm.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/transition.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/typeRegistry.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/unknownSchema.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/vectorIndexing.h
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/include/opentimelineio/version.h
-- Installing: /opentimelineio/opentimelineio.lib
-- Installing: /opentimelineio/opentimelineio.dll
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/OpenTimelineIOConfig.cmake
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/OpenTimelineIOConfig-release.cmake
-- Installing: /opentimelineio/_opentime.pyd
-- Installing: /opentimelineio/_otio.pyd
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/python/bin
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/python/bin/otiocat.bat
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/python/bin/otioconvert.bat
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/python/bin/otiostat.bat
-- Installing: C:/Daten/Olive/Deps/OpenTimelineIO/build7/install/python/bin/otiopluginfo.bat
```